examples/hypervisor_managers/update_resource.tf

```diff
@@ -0,0 +1,39 @@
+provider "oneview" {
+  ov_username   = var.username
+  ov_password   = var.password
+  ov_endpoint   = var.endpoint
+  ov_sslverify  = var.ssl_enabled
+  ov_apiversion = 2600
+  ov_ifmatch    = "*"
+}
+
+variable "hm_endpoint" {
+  type        = string
+  description = "Hypervisor Manager IP"
+  default     = "172.18.13.11" #"<HM-IP>"
+}
+
+variable "hm_username" {
+  type        = string
+  description = "Hypervisor Manager Username"
+  default     = "dcs" #"<HM-Username>"
+}
+
+# Update the resource Post applying main.tf  
+resource "oneview_hypervisor_manager" "HypervisorManager" {
```

Comment:
for update, we need to send force parameter from 2600, even if user is using lesser API Version, golang will expect empty string as an attribute.

Reply:
Done